### PR TITLE
FreeBSD yaml update

### PR DESCRIPTION
**Pull request by thiagoftsm**
##### Summary
Add description for `freebsd`  plugin.

##### Test Plan

1. Take a look in CI and grammar.

##### Additional Information

<details> <summary>For users: How does this change affect me?</summary>
Describe the PR affects users: 
- Which area of Netdata is affected by the change? documentation, integration and dashboard.
- Can they see the change or is it an under the hood? If they can see it, where? no
- How is the user impacted by the change? a plugin well documented.
- What are there any benefits of the change? A more clear information for our users.
</details>

### CODE REVIEW

File: collectors/freebsd.plugin/multi_metadata.yaml [hunk 1]
```diff
@@ -38,14 +40,19 @@ modules:
         list: []
       configuration:
         file:
-          name: ""
-          description: ""
+          name: "netdata.conf"
+          section_name: "plugin:freebsd"
```
Comment (Ancairon) [inline collectors/freebsd.plugin/multi_metadata.yaml:44]:
```suggestion
          section_name: "[plugin:freebsd]"
```

File: collectors/freebsd.plugin/multi_metadata.yaml [hunk 2]
```diff
@@ -132,14 +139,27 @@ modules:
         list: []
       configuration:
         file:
-          name: ""
+          name: "netdata.conf"
+          section_name: "plugin:freebsd:vm.vmtotal"
```
Comment (Ancairon) [inline collectors/freebsd.plugin/multi_metadata.yaml:143]:
```suggestion
          section_name: "[plugin:freebsd:vm.vmtotal]"
```

File: collectors/freebsd.plugin/multi_metadata.yaml [hunk 3]
```diff
@@ -221,14 +241,18 @@ modules:
         list: []
       configuration:
         file:
-          name: ""
-          description: ""
+          name: "netdata.conf"
+          description: "plugin:freebsd"
```
Comment (Ancairon) [inline collectors/freebsd.plugin/multi_metadata.yaml:245]:
```suggestion
          description: "[plugin:freebsd]"
```

File: collectors/freebsd.plugin/multi_metadata.yaml [hunk 4]
```diff
@@ -331,14 +355,19 @@ modules:
         list: []
       configuration:
         file:
-          name: ""
-          description: ""
+          name: "netdata.conf"
+          section_name: "plugin:freebsd"
```
Comment (Ancairon) [inline collectors/freebsd.plugin/multi_metadata.yaml:359]:
```suggestion
          section_name: "[plugin:freebsd]"
```

File: collectors/freebsd.plugin/multi_metadata.yaml [hunk 5]
```diff
@@ -403,14 +432,19 @@ modules:
         list: []
       configuration:
         file:
-          name: ""
-          description: ""
+          name: "Config options"
+          section_name: "plugin:freebsd"
```
Comment (Ancairon) [inline collectors/freebsd.plugin/multi_metadata.yaml:436]:
```suggestion
          section_name: "[plugin:freebsd]"
```

File: collectors/freebsd.plugin/multi_metadata.yaml [hunk 6]
```diff
@@ -475,14 +509,19 @@ modules:
         list: []
       configuration:
         file:
-          name: ""
-          description: ""
+          name: "netdata.conf"
+          section_name: "plugin:freebsd"
```
Comment (Ancairon) [inline collectors/freebsd.plugin/multi_metadata.yaml:513]:
```suggestion
          section_name: "[plugin:freebsd]"
```

File: collectors/freebsd.plugin/multi_metadata.yaml [hunk 7]
```diff
@@ -553,14 +592,19 @@ modules:
...
```
Comment (Ancairon) [inline collectors/freebsd.plugin/multi_metadata.yaml:596]:
```suggestion
          section_name: "[plugin:freebsd]"
```

File: collectors/freebsd.plugin/multi_metadata.yaml [hunk 8]
```diff
@@ -625,14 +669,19 @@ modules:
...
```
Comment (Ancairon) [inline collectors/freebsd.plugin/multi_metadata.yaml:673]:
```suggestion
          section_name: "[plugin:freebsd]"
```

File: collectors/freebsd.plugin/multi_metadata.yaml [hunk 9]
```diff
@@ -697,14 +746,19 @@ modules:
...
```
Comment (Ancairon) [inline collectors/freebsd.plugin/multi_metadata.yaml:750]:
```suggestion
          section_name: "[plugin:freebsd]"
```

File: collectors/freebsd.plugin/multi_metadata.yaml [hunk 10]
```diff
@@ -775,14 +829,19 @@ modules:
...
```
Comment (Ancairon) [inline collectors/freebsd.plugin/multi_metadata.yaml:833]:
```suggestion
          section_name: "[plugin:freebsd]"
```

File: collectors/freebsd.plugin/multi_metadata.yaml [hunk 11]
```diff
@@ -853,14 +912,19 @@ modules:
...
```
Comment (Ancairon) [inline collectors/freebsd.plugin/multi_metadata.yaml:916]:
```suggestion
          section_name: "[plugin:freebsd]"
```

File: collectors/freebsd.plugin/multi_metadata.yaml [hunk 12]
```diff
@@ -957,14 +1021,19 @@ modules:
...
```
Comment (Ancairon) [inline collectors/freebsd.plugin/multi_metadata.yaml:1025]:
```suggestion
          section_name: "[plugin:freebsd]"
```

File: collectors/freebsd.plugin/multi_metadata.yaml [hunk 13]
```diff
@@ -1035,14 +1104,19 @@ modules:
...
```
Comment (Ancairon) [inline collectors/freebsd.plugin/multi_metadata.yaml:1108]:
```suggestion
          section_name: "[plugin:freebsd]"
```

File: collectors/freebsd.plugin/multi_metadata.yaml [hunk 14]
```diff
@@ -1111,14 +1185,19 @@ modules:
...
```
Comment (Ancairon) [inline collectors/freebsd.plugin/multi_metadata.yaml:1189]: Please add the brackets to the rest, from checking diskspace.plugin and how it renders, I understand we need brackets in these

**Review by Ancairon — COMMENTED** [review]
Everything has no metrics description for scopes.. as I said in another PR there is boilerplate we add for global and other scopes, take a look at python plugin and charts.d

**Review by sashwathn — APPROVED** [review]

**Pull request merged**